### DISCUSSION TITLE
Curious case of concept map smartcropping

### DISCUSSION THREAD

**Discussion by sergeevabc** [libvips/libvips#3821] [Help wanted]
I am looking for a way to crop the following image (1549×1272) as follows (600×100).
Trim seems to be the key, but I'm not sure how to get it right with `vipsthumbnail`.

* [Original](https://github.com/libvips/libvips/assets/373718/8204e539-9e49-44b6-959c-477fb83ad795)
* [Current smartcrop](https://github.com/libvips/libvips/assets/373718/c36d2d3d-5b2c-4cab-b1e7-a2cc6bb7425d) via `--smartcrop entropy -s 600x100`
* [Expected smartcrop](https://wsrv.nl/?url=https://pic.maxiol.com/images2/1705634494.774319425.conceptmap.png&w=600&&height=100&trim=100&fit=cover&a=entropy) via third-party service with additional trim set to 100

**Comment by jcupitt** [comment]
Hi again, `smartcrop` is not very smart and it's unlikely to pick the area you want on this image.

Did you see https://github.com/libvips/libvips/discussions/3767 ? Though that's for the visual attention smartcrop.

**Comment by sergeevabc** [comment]
Dear @kleisauke, @kbsali and @andrieslouw, since Wsrv.nl is involved, could you be so kind to tell what exactly does your service do when `&trim=100` is called? Without that flag, the output of your service and `vipsthumbnail --smartcrop entropy` are the same here. But it's that flag that brings the very magic I'm looking for.
